### PR TITLE
15_1_balloon

### PR DESCRIPTION
**Pull request by AaronDabney**
## Balloon 🎈
- Added `balloonHandler` function
- `balloonHandler` is added to first element with "balloon"` ID
- Balloon is centered via style tag to any browser dimension
- `balloonHandler` event listener is removed from window by `explosion` function

https://github.com/AaronDabney/InterviewPrep/assets/172078271/2a23face-c6c0-49d8-80b5-a56f5ed9e9d7

### CODE REVIEW

File: eloquentjs/15_1_balloon/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Balloon</title>
+    <style>
+        body {
+            margin: 0%;
+            padding: 0%;
+            height: 100vh;
+            align-content: center;
+            background-color: rgb(49, 60, 87);
+        }
```
Comment (CalHinshaw) [inline eloquentjs/15_1_balloon/index.html:14]: a line break is standard

File: eloquentjs/15_1_balloon/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+let balloonSize = 1;
+let explosionThreshold = 18;
+
+let timeout;
+
+let balloonHandler = event => {
+    if (event.key === 'ArrowUp') {
+        clearTimeout(timeout);
+
+        timeout = setTimeout(() =>{
```
Comment (CalHinshaw) [inline eloquentjs/15_1_balloon/index.js]: I don't think you should use setTimeout here. It would be better to avoid that complexity.

**Review by CalHinshaw — CHANGES_REQUESTED** [review]
This is a great start!

**Comment by AaronDabney** [comment]
https://github.com/AaronDabney/InterviewPrep/assets/172078271/7d72c98f-eb95-414a-9551-8e000cd5dfa1

**Review by CalHinshaw — APPROVED** [review]
This looks a lot simpler, nice work.

**Pull request merged**